tecan_od_analyzer/tecan_od_analyzer.py

```diff
@@ -289,15 +287,13 @@ def compensation_lm(cor_df, df_gr) : #done
 		lm_eq.append(popt)
 		
 		ax = fig.add_subplot(math.ceil(len(unique_bioshaker)), math.ceil(len(unique_bioshaker)/2), shaker+1)
-		#ax.get_xaxis().set_visible(True)
-		#ax.get_yaxis().set_visible(True)
 		ax.plot(sub_cor_df["time_hours"], sub_cor_df["Correlation"], "o", label="Empirical data", markerfacecolor='skyblue', markeredgecolor="dodgerblue", alpha= 0.5)
 		ax.plot(sub_cor_df["time_hours"], linear(sub_cor_df ["time_hours"], *popt), "b-", label="Linear model", color = "darkred")
-		ax.set_ylabel('Volume loss correlation', labelpad=10)
+		ax.set_ylabel('Correlation', labelpad=10)
```

Comment:
In general, it is good to specify the units for all axes labels.  If the units are arbitrary, please us (au) for arbitrary units.  If units are not applicable, please use (na).